src/components/Compare/columns.tsx

```diff
@@ -0,0 +1,191 @@
+import { Metric } from 'common/metricEnum';
```

Comment:
Sorry, I renamed this file and then edited it so much that github isn't showing the diff against the old `columns.ts` file.  But if you just look at the 2nd commit in my PR (https://github.com/covid-projections/covid-projections/pull/3885/commits/53e036d9aabb6945f99ab071ce19aae1c01f2d6f) then you can see just the diffs (though admittedly I changed a lot 😁 ).